File: sample/src/main/java/com/aiqcomponents/sample/MainActivity.java
package com.aiqcomponents.sample;

import android.graphics.Color;
import android.os.Bundle;
import android.os.Handler;
import android.os.Looper;
import android.widget.Button;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;
import androidx.vectordrawable.graphics.drawable.VectorDrawableCompat;

import com.aiqcomponents.sample.databinding.ActivityMainBinding;
import com.aiqcomponents.sample.models.Country;
import com.aiqcomponents.sample.models.Region;
import com.aiqfome.aiqcomponents.adapters.BaseItem;
import com.aiqfome.aiqcomponents.adapters.IconItem;
import com.aiqfome.aiqcomponents.progressbutton.DrawableButton;
import com.aiqfome.aiqcomponents.progressbutton.DrawableButtonExtensionsKt;
import com.aiqfome.aiqcomponents.selector.SelectorController;
import com.aiqfome.aiqcomponents.textinput.TextInputController;

import java.util.ArrayList;
import java.util.List;

import kotlin.Unit;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);
    }
}

File: sample/src/main/java/com/aiqcomponents/sample/MainActivity.kt
package com.aiqcomponents.sample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.fragment.NavHostFragment
import androidx.navigation.fragment.findNavController
import androidx.navigation.ui.NavigationUI

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)
        val navHost =
                supportFragmentManager.findFragmentById(R.id.nav_host_fragment) as NavHostFragment
        val navController = navHost.findNavController()
        NavigationUI.setupActionBarWithNavController(this, navController)
    }

    override fun onSupportNavigateUp(): Boolean {
        super.onBackPressed()

        return true
    }
}